tests/arch/arm64/arm64_smc_call/testcase.yaml

```diff
@@ -0,0 +1,11 @@
+tests:
+  arch.arm64.smc_call.smc:
+    arch_allow: arm
+    tags: arm drivers userspace
+    filter: CONFIG_BUILD_WITH_TFA
+  arch.arm64.smc_call.hvc:
+    arch_allow: arm
+    tags: arm drivers userspace
+    filter: CONFIG_BUILD_WITH_TFA
+    extra_configs:
+      - CONFIG_SMC_CALL_USE_HVC=y
```

Comment:
These tests are being skipped by every platform (not run at all) because `BUILD_WITH_TFA` is not selected by any platforms.